mlir/include/mlir/Dialect/Rock/IR/RockOps.td

```diff
@@ -1048,6 +1048,10 @@ def Rock_BlockwiseGemmOp:
     Arguments<(ins MemRefRankOf<GemmInputTypes, [3]>:$matrixA,
                    MemRefRankOf<GemmInputTypes, [3]>:$matrixB,
                    MemRefRankOf<GemmOutputTypes, [2]>:$matrixC,
+                   I32Attr:$inMPerThread,
+                   I32Attr:$inNPerThread,
+                   UnitAttr:$rotateMWithK,
+                   UnitAttr:$rotateNWithK,
```

Comment:
We might need to describe these flags in the description for others who haven;t followed this PR.